tests/integration/test_sharepoint.py

```diff
@@ -11,32 +11,72 @@
 from viadot.tasks.sharepoint import SharepointToDF
 
 
-def get_url():
+def get_url() -> str:
+    """
+    Function to get file URL.
+
+    Returns:
+        str: File URL.
+    """
     return local_config["SHAREPOINT"].get("url")
 
 
 @pytest.fixture(scope="session")
 def sharepoint():
+    """
+    Fixture for creating a Sharepoint class instance.
+    The class instance can be used within a test functions to interact with Sharepoint.
+    """
     s = Sharepoint()
     yield s
 
 
 @pytest.fixture(scope="session")
 def FILE_NAME(sharepoint):
```

Comment:
other fixtures are named in lowercase, maybe change it to be more consistent

Reply:
Done